crates/ra_ide/src/syntax_highlighting.rs

```diff
@@ -361,7 +361,9 @@ fn highlight_element(
         }
 
         // Highlight references like the definitions they resolve to
-        NAME_REF if element.ancestors().any(|it| it.kind() == ATTR) => return None,
+        NAME_REF if element.ancestors().any(|it| it.kind() == ATTR) => {
+            Highlight::from(HighlightTag::Function) | HighlightModifier::Attribute
```

Comment:
I guess, this gives a nicer looks, but semantically this feels wrong. Like, it's not a function, it's a part of attribute syntax.

Maybe we should introduce something like `AttrKey` here? Not that it really matters though.